c10/util/complex_math.cpp

```diff
@@ -44,6 +44,22 @@ c10::complex<T> compute_csqrt(const c10::complex<T>& z) {
   auto t = std::sqrt((-z.real() + std::abs(z)) * half);
   return c10::complex<T>(half * std::abs(z.imag() / t), std::copysign(t, z.imag()));
 }
+
+template<typename T>
+c10::complex<T> compute_cacos(const c10::complex<T>& z) {
```

Comment:
minor: Would be nice if the reference formula from the description is commented here.